### DISCUSSION TITLE
[2024년 03월 01일] 회의록

### DISCUSSION THREAD

**Discussion by Subak-Uncle** [cca-ffodregamdi/running-hi-back-v2#30] [dailyscrum]
# 📋 회의 내용
- 오후 10시에 모여서 와이어 프레임 제작
- 아이디어 생각해오기
- 금, 토, 일 개발 내용은 월요일 날 공유
- 키워드 도메인 방식 논의
    - 중간 객체 없이 키워드 테이블에
        - KeywordNo PK
        - MemberPostNo FK
        - AdminPostNo FK
        - KeywordName
    - 중간 객체 키워드 게시물 테이블
        
        [TBL_KEYWORD_TABLE]
        
        - KeywordNo FK
        - MemberPostNo FK
        - AdminPostNo FK
        
        [TBL_KEYWORD]
        
        - KeywordNo PK
        - KeywordName
  

# 🦋 알게된 사실 or 참고 자료 
[[Legacy DB의 JPA Entity Mapping (복합키 매핑 편) | 우아한형제들 기술블로그](https://techblog.woowahan.com/2595/)](https://techblog.woowahan.com/2595/)

**Comment by silverpoodle** [comment]
정리 감사합니다. 깔끔티비!
